flutter/lib/app_constants.dart

```diff
@@ -3,6 +3,8 @@ import 'package:flutter/material.dart';
 const OFFICIAL_BUILD =
     bool.fromEnvironment('official_build', defaultValue: false);
 
+const FAST_MODE = bool.fromEnvironment('fast-mode', defaultValue: false);
```

Comment:
Please rename `fast-mode` to `fast_mode`, or rename `official_build` to `official-build`, so we are consistent in using either underscore or hyphen.

Reply:
Done. I renamed `official_build` to `official-build`